tox.ini

```
@@ -0,0 +1,18 @@
+[tox]
+envlist = py35,py36,pypy,pypy3
```

Python 2 is no longer supported, and 3.7 and 3.8 are supported.

```suggestion
envlist = py35,py36,py37,py38,pypy3
```